src/Build/parse/generate_jupyter.py

```diff
@@ -0,0 +1,176 @@
+# Displaying all the inouts in one widget
+import os
+import nbformat as nbf
+from ipywidgets import widgets, VBox
+from IPython.display import display
+from parse import main as parse_config
+
+
+def preprocess_config(config):
+    supported_input_types = {'Files', 'Radio', 'Number', 'Textbox', 'Checkbox'}
+    inputs = [input_conf for input_conf in config[1] if input_conf['type'] in supported_input_types]
+    outputs = config[2]
+    exec_function = config[3]
+    return inputs, outputs, exec_function
+
+def create_code_cell(content):
+    return nbf.v4.new_code_cell(content)
+
+def create_markdown_cell(content):
+    return nbf.v4.new_markdown_cell(content)
+
+def create_widget_code(inputs, outputs, exec_function):    
+    newest_code = f"""
```

Comment:
We should put the contents of `newest_code` in a separate file, and then just read it in here. That way we get nice syntax highlighting, etc.

Reply:
This file was the first iteration/attempt to generate the jupyter notebook. But now, it's been changed to Jinja2 Template code and the whole logic is separated in `generate.py` and `jupyter_template.py.j2`. Hence, it no longer exists!

Reply:
This file does exist in this PR, currently. Please push the commit that deletes it, then we can resolve.